lib/group_site_manager.py

```diff
@@ -481,6 +484,50 @@ def _upload_dtmd(self, numbers: List[str]):
     finally:
       driver.quit()
 
+  def _login_emb(self) -> WebDriver:
+    group_config = self.config['groups']['embdeals']
+    username = group_config['username']
+    password = group_config['password']
+    driver = self.driver_creator.new()
+
+    # the website is terrible, give it leeway
```

Comment:
is the emb website terrible like oaks is? i would believe it

Reply:
Yes, incredibly slow.